creator-node/src/routes/nodeSync.js

```diff
@@ -199,6 +200,14 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint) {
         throw new Error(`Malformed response received from ${creatorNodeEndpoint}. "walletPublicKey" property not found on CNodeUser in response object`)
       }
       const fetchedWalletPublicKey = fetchedCNodeUser.walletPublicKey
+      const myCnodeEndpoint = await middlewares.getOwnEndpoint(req)
+      const userReplicaSet = await middlewares.getCreatorNodeEndpoints(req, fetchedWalletPublicKey)
```

Comment:
short circuit + error here if list is empty?

Reply:
i think the only way this would be empty is if a primary calls a sync on itself and there's no user metadata node for some reason?

Reply:
both of these functions `getOwnEndpoint()` and `getCreatorNodeEndpoints()` can throw an error or return empty and we should handle those cases here